habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -293,6 +293,7 @@ class OracleNavActionConfig(ActionConfig):
     ang_speed: float = 10.0
     allow_dyn_slide: bool = True
     allow_back: bool = True
+    # A value of -1.0 means we will get as close to the object as possible.
```

Comment:
How do we define as close to the object as possible? Maybe a short second sentence to clarify.

Reply:
This is addressed in this PR actually: https://github.com/facebookresearch/habitat-lab/pull/1209